orion/pkg/controllers/location_controller.go

```diff
@@ -88,3 +90,20 @@ func DeleteLocation(c *gin.Context) {
 	}
 	return
 }
+
+func PublishLocations(c *gin.Context) {
+	// Incoming JSON
+	var locIdsJson []string
+	c.BindJSON(&locIdsJson)
+
+	errors := services.LocationService.Publish(locIdsJson)
+	if len(errors) > 0 {
+		for _, err := range errors {
+			c.Error(err.Error)
+			c.String(http.StatusInternalServerError, err.Error.Error())
```

Comment:
I don't think the front-end should get a bunch of errors when they only send one request.